src/components/ContactPage/ContactPage.tsx

```diff
@@ -1,73 +1,129 @@
 import * as React from "react";
+import {Modal} from "../Modal";
+import {ContactPageState} from "./ContactPageState";
 
-export class ContactPage extends React.Component<undefined, undefined> {
+import {OnMobile, OnMobileTablet, OnDesktop, OnTablet} from "../../helpers/Breakpoints";
 
-    public render() {
+// tslint:disable
+export class ContactPage extends React.Component<undefined, ContactPageState> {
+
+    public state = {
+        isModalOpen: false,
+        isOnSubmit: false,
+    };
+
+    public render(): JSX.Element {
         return (
             <section className="section section-partnership">
                 <h2 className="section-title">Партнерство</h2>
                 <div className="section-half half-first">
-                    <h4 className="section-subtitle">
-                        Свяжитесь с нами
-                        <span className="section-subtitle_reduced">
+                    <OnDesktop>
+                        <h4 className="section-subtitle">
+                            Свяжитесь с нами
+                            <span className="section-subtitle_reduced">
                             или укажите свои контактные данные в форме ниже.
                             Наши специалисты ответят на все ваши вопросы.
                         </span>
-                    </h4>
-                    <form className="form">
-                        <div className="form__group form__group_has-error">
-                            <input type="text" className="form__control" placeholder="Ваше имя"/>
-                            <span className="form__control_underline"/>
-                            <span className="form__error-text">текст ошибки</span>
-                        </div>
-                        <div className="form__group_inline">
-                            <div className="form__group">
-                                <input type="tel" className="form__control" placeholder="Телефон"/>
-                                <span className="form__control_underline"/>
+                        </h4>
+                        {this.submitForm}
+                    </OnDesktop>
+                </div>
+                <div className="section-half half-second">
+                    <div className="contact-info">
+                        <OnDesktop>
```

Comment:
This
`<a href="tel:+380660249402" className="contact-info__link">380 66 024-94-02</a>`
and mail links are duplicated. Will we change a lot of lines in case of changing phone?
Phone and email should be moved to config.